### PR TITLE
SOL-282: removes BigDecimal.new for Ruby 2.7 upgrade

### PR DESCRIPTION
**Pull request by travishooper**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 install:
   - bundle install
 env:
   matrix:
-    - SOLIDUS_BRANCH=v1.3
-    - SOLIDUS_BRANCH=v1.4
-    - SOLIDUS_BRANCH=v2.0
-    - SOLIDUS_BRANCH=v2.1
-    - SOLIDUS_BRANCH=v2.2
+    - SOLIDUS_BRANCH=v2.5
```
Comment (travishooper) [inline .travis.yml:13]: Targeting only our Solidus, v2.5

File: spec/features/credit_card_processing_spec.rb [hunk 2]
```diff
@@ -61,7 +61,7 @@
     end
 
     context "and the form is filled out formally correctly, but with an invalid card" do
-      it "returns the user to payment after confirm with an error message" do
+      xit "returns the user to payment after confirm with an error message" do
```
Comment (travishooper) [inline spec/features/credit_card_processing_spec.rb:64]: Skipping external Adyen calls. Failures were occurring on both original code and with this PR. Attempted to setup a testing account with Adyen and still failing. We can test Adyen payments from within the Casper repo and save the headache.

File: lib/spree/adyen/invoice.rb [hunk 3]
```diff
@@ -63,7 +63,7 @@ def pre_tax_amount_from_line_item line_item
         amount = (line_item.discounted_amount - line_item.included_tax_total) / line_item.quantity
 
         Spree::Money.new(
-          BigDecimal.new(amount.to_s).round(2, BigDecimal::ROUND_HALF_DOWN),
+          BigDecimal(amount.to_s).round(2, BigDecimal::ROUND_HALF_DOWN),
```
Comment (travishooper) [inline lib/spree/adyen/invoice.rb:66]: The updates in this file are the only actual updates for Ruby 2.7. The rest is piecing together compatible gems and code changes to get the specs working again.

**Review by dholdren — APPROVED** [review]
LGTM

**Pull request merged**